api/contract.go

```diff
@@ -49,6 +49,34 @@ type getFilterResponse struct {
 	model.Filter
 }
 
+// createFilterResponse is the response body for POST /filters
+type createFilterOutputResponse struct {
+	model.FilterOutput
+}
+
+// createFilterRequest is the request body for POST /filters
+type createFilterOutputsRequest struct {
```

Comment:
Should be `createFilterOutputRequest` singular, and comment is incorrect. JSON tags could be lined up too.